src/lib/runtime_config/runtime_config.ml

```diff
@@ -1253,8 +1252,21 @@ let gen =
   }
 
 let ledger_accounts (ledger : Mina_base.Ledger.Any_ledger.witness) =
-  Mina_base.Ledger.Any_ledger.M.to_list ledger
-  |> Async.Deferred.map ~f:(map_results ~f:Accounts.Single.of_account)
+  let open Async.Deferred.Result.Let_syntax in
+  let yield = Async_unix.Scheduler.yield_every ~n:100 |> Staged.unstage in
+  let%bind accounts =
+    Mina_base.Ledger.Any_ledger.M.to_list ledger
+    |> Async.Deferred.map ~f:Result.return
+  in
+  let%map accounts =
+    deferred_list_fold ~init:[]
+      ~f:(fun acc el ->
+        let%bind () = yield () |> Async.Deferred.map ~f:Result.return in
```

Comment:
```suggestion
        let%bind () = yield () >>| Result.return
```


It'd require replace `let%bind` above with `let%bind.Deferred.Result` and removal of `open Deferred.Result.Let_syntax`

I believe it's cleaner, though it's a NIT.

Reply:
Proposed a fix in [29131a6](https://github.com/MinaProtocol/mina/pull/14889/commits/29131a603048b1f96e55205eeab45cd738d97bdd).

Reply:
That fix wasn't applied here

Reply:
Sorry, applied in 6f44a34e98.